nodes/mod.go

```diff
@@ -288,3 +288,33 @@ func downloadModVersion(c echo.Context) error {
 
 	return c.Redirect(302, storage.GenerateDownloadLink(version.Key))
 }
+
+func downloadModVersionArch(c echo.Context) error {
```

Comment:
This function appears to lack a doc comment, whereas all others in this file have one

Reply:
https://github.com/satisfactorymodding/smr-api/pull/12/commits/5e5146206a7fe78acf4f87620417a535e6a9a579